iris-mpc-gpu/src/server/actor.rs

```diff
@@ -523,7 +532,27 @@ impl ServerActor {
                 batch,
                 return_channel,
             } = job;
-            let _ = self.process_batch_query(batch, return_channel);
+
+            match self.process_batch_query(batch.clone(), Orientation::Mirror, None) {
```

Comment:
Actually there is no need to do the mirror stuff for both Reset and Reauth requests. So those could be kicked out of the batch for the mirror run. Certainly not worth the hassle now, but probably something to keep in mind for the future.